azkaban-web-server/src/main/java/azkaban/webapp/WebMetrics.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.webapp;
+
+import azkaban.metrics.MetricsManager;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+
+/**
+ * This singleton class WebMetrics is in charge of collecting varieties of metrics
+ * from azkaban-web-server modules.
+ */
+public enum WebMetrics {
+  INSTANCE;
+
+  private MetricRegistry registry;
+
+  private Meter webGetCall;
+  private Meter webPostCall;
+
+  // How long does user log fetch take when user call fetch-log api.
+  private AtomicLong _logFetchLatency = new AtomicLong(0L);
+
+  WebMetrics() {
+    registry = MetricsManager.INSTANCE.getRegistry();
+    setupAllMetrics();
+  }
+
+  private void setupAllMetrics() {
+    webGetCall = addMeter("Web-Get-Call-Meter");
+    webPostCall = addMeter("Web-Post-Call-Meter");
+    addLongGauge("fetchLogLatency", _logFetchLatency);
+  }
+
+  public Meter addMeter(String name) {
```

Comment:
Consider using a utility method to share the implementation of the addMeter method? Do we need to add additional methods for other metrics in the future? If so, the code duplication will increase.